otter/integration/lib/autoscale.py

```diff
@@ -20,6 +21,13 @@
     transient_errors_except,
 )
 
+pp = pprint.PrettyPrinter(indent=4)
+# Get vs dict lookup because it will return 0 if not found, not throw an
```

Comment:
Non-blocking - this comment seems superfluous, since the next line of code pretty much says this exactly.